index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!-- <p> Hello World, we are software developers!</p> -->
+    
+    <main id="main">
+      <h1 id="title">Doberman dogs are beatiful!</h1>
+      <div id="img-div">
+        <div id="image-container">
+          <img id="image" src="https://i.blogs.es/1d6dd6/istock-1324413613/840_560.jpeg" alt="">
```

Comment:
For SEO  it is important to add a name to the images ⚡you can do it by filling in alt="name of image" 👍🏾